src/components/Auth.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from "react";
+import imgur from "../assets/Imgur_logo.svg";
+import logo from "../svgs/logo.js";
+import close from "../svgs/close";
+import FloatingButton from "./FloatingButton";
+import AuthButton from "./AuthButton";
+
+class Auth extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+  render() {
+    return (
+      <section className="auth">
+        <img src={imgur} alt="imgur" />
+
+        <AuthButton>Connect with Imgur</AuthButton>
+
+        <FloatingButton handleClick={this.props.history.goBack}>
```

Comment:
Is this required?

Reply:
`<FloatingButton>` is not required anymore, will remove that. 
`<AuthButton>` is also not required as of now, but we can make a new `<Button>` component for using the same styles OR we can use class to style the buttons.